Rubberduck.CodeAnalysis/Inspections/Concrete/ReadOnlyPropertyAssignmentInspection.cs

```diff
@@ -102,7 +102,7 @@ protected override bool IsResultReference(IdentifierReference reference, Declara
         {
             if (!reference.Declaration.DeclarationType.HasFlag(DeclarationType.Property)
                 //Ignore expressions found within Property declaration contexts
-                || reference.Declaration.Context.Contains(reference.Context))
+                || (reference.Declaration.Context?.Contains(reference.Context) ?? false))
             {
                 return false;
             }
```

Comment:
FYI - the workbook I was using for testing my last PR generated an NRE at this exact line of code.  As a result, I've started work on a PR that updates the `ParserRuleContextExtension` public methods that do not check for `null` arguments like `ParserRuleContext.Contains(...)`.

The PR will avoid this NRE scenario (and possibly others) that are occurring 'out in the wild'.  And of course, there is no harm in leaving this null check on the client-side as well.